packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -172,19 +171,6 @@ export type Thenable = {
   then(resolve: () => mixed, reject?: () => mixed): mixed,
 };
 
-// Intentionally not named imports because Rollup would
```

Comment:
Mind keeping these?

Reply:
They're not being used in this module anymore. Do you mean comment these lines out?

Reply:
I'll comment them out so we don't forget next time we import Scheduler in this module